NoteSalihMVVM/Scenes/Profile/ProfileViewController.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  ProfileViewController.swift
+//  NoteSalihMVVM
+//
+//  Created by Mehmet Salih ÇELİK on 3.03.2022.
+//
+
+final class ProfileViewController: BaseViewController<ProfileViewModel> {
+    
+    private let stackView = UIStackViewBuilder()
+        .spacing(15)
+        .axis(.vertical)
+        .build()
+    private let fullNameTextField = AuthTextField()
+    private let emailTextField = AuthTextField()
+    private let saveButton = AuthButton()
+    private let changePasswordButton = UIButtonBuilder()
+        .build()
+    private let signOutButton = UIButtonBuilder()
+        .build()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addSubViews()
+        configureContents()
+        setLocalize()
+        subscribeViewModel()
+        viewModel.getUser()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationItem.setHidesBackButton(true, animated: true)
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        navigationItem.setHidesBackButton(false, animated: true)
+    }
+}
+
+// MARK: - UILayout
+extension ProfileViewController {
+    
+    private func addSubViews() {
+        view.addSubview(stackView)
+        stackView.edgesToSuperview(excluding: .bottom, insets: .init(top: 25, left: 25, bottom: 25, right: 25), usingSafeArea: true)
+        stackView.addArrangedSubview(fullNameTextField)
+        stackView.addArrangedSubview(emailTextField)
+        stackView.addArrangedSubview(saveButton)
+        stackView.addArrangedSubview(changePasswordButton)
+        stackView.addArrangedSubview(signOutButton)
+    }
+}
+
+// MARK: - Configure & SetLocalize
+extension ProfileViewController {
+    
+    private func configureContents() {
+        configureNavigation()
+        configureSaveButton()
+        configureChangePasswordButton()
+        configureSignOutButton()
+    }
+    
+    private func configureNavigation() {
+        let leftIcon = UIBarButtonItem(image: .icHamburger, style: .plain, target: self, action: #selector(leftIconTapped))
+        navigationItem.leftBarButtonItems = [leftIcon]
+    }
+    
+    private func configureSaveButton() {
+        saveButton.addTarget(self, action: #selector(saveButtonTapped), for: .touchUpInside)
```

Comment:
tek satır için bölmeye gerek yok configure içine direk yazılabilir

Reply:
fix